packages/components/src/components/contract-card/contract-card-items/contract-card-header.tsx

```diff
@@ -71,12 +71,14 @@ const ContractCardHeader = ({
     const is_mobile = isMobile();
     const is_turbos = isTurbosContract(contract_type);
     const is_lookbacks = isLookBacksContract(contract_type);
+    const is_multipliers = !!multiplier && !is_lookbacks;
+    const is_high_low = isHighLow({ shortcode });
 
     const contract_type_list_info = React.useMemo(
         () => [
             {
-                is_param_displayed: multiplier && !is_lookbacks,
-                displayed_param: `x${multiplier}`,
+                is_param_displayed: is_multipliers,
```

Comment:
I think it shouldn't combine lookbacks under multipliers here

Reply:
Both have multiplier (e.g. x100, x200 ). So we need to determine somehow if this contract is Multipliers or LookBacks. Logic with checking  was there before, I just refactored to make in clear

Reply:
I refactored logic and added check isMultipliersContract, so it's not combined anymore